Gemfile

```diff
@@ -22,7 +22,7 @@ gem "default_value_for",              "~>3.0.2"
 gem "elif",                           "=0.1.0",    :require => false
 gem "fast_gettext",                   "~>1.1.0"
 gem "fog-google",                     "~>0.3.0",   :require => false
-gem "fog-vcloud-director",            "~>0.1.1",   :require => false
+gem "fog-vcloud-director",            "~>0.1.3",   :require => false
```

Comment:
I had to update fog-vcloud-director with a method to retrieve a single catalog item: https://github.com/xlab-si/fog-vcloud-director/commit/4d01d1ded249c77b64921a93b38f6c61ef23f6f6. This is used in `Template` to retrieve the provider object.